docker/kubernetes-tentacle/scripts/configure-tentacle.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+set -e
+
+if [[ "$ACCEPT_EULA" != "Y" ]]; then
+    echo "ERROR: You must accept the EULA at https://octopus.com/company/legal by passing an environment variable 'ACCEPT_EULA=Y'"
+    exit 1
+fi
+
+# Tentacle Docker images only support once instance per container. Running multiple instances can be achieved by running multiple containers.
+instanceName=Tentacle
+configurationDirectory=/etc/octopus
+applicationsDirectory=/home/Octopus/Applications
+internalListeningPort=10933
+
+mkdir -p $configurationDirectory
+mkdir -p $applicationsDirectory
+
+if [ ! -f /usr/bin/tentacle ]; then
+    ln -s /opt/octopus/tentacle/Tentacle /usr/bin/tentacle
+fi
+
+function getPublicHostName() {
+    if [[ "$PublicHostNameConfiguration" == "PublicIp" ]]; then
+        curl https://api.ipify.org/
+    elif [[ "$PublicHostNameConfiguration" == "FQDN" ]]; then
+        hostname --fqdn
+    elif [[ "$PublicHostNameConfiguration" == "ComputerName" ]]; then
+        hostname
+    else
+        echo $CustomPublicHostName
+    fi
+}
+
+function validateVariables() {
+    if [[ -z "$ServerApiKey" && -z "$BearerToken" ]]; then
+        if [[ -z "$ServerPassword" || -z "$ServerUsername" ]]; then
+            echo "Please specify either an API key, a Bearer Token or a username/password with the 'ServerApiKey' or 'ServerUsername'/'ServerPassword' environment variables" >&2
+            exit 1
+        fi
+    fi
+
+    if [[ -z "$ServerUrl" ]]; then
+        echo "Please specify an Octopus Server with the 'ServerUrl' environment variable" >&2
+        exit 1
+    fi
+
+    if [[ -z "$TargetEnvironment" ]]; then
+        echo "Please specify an environment name with the 'TargetEnvironment' environment variable" >&2
+        exit 1
+    fi
+
+    if [[ -z "$TargetRole" ]]; then
+        echo "Please specify a role name with the 'TargetRole' environment variable" >&2
+        exit 1
+    fi
```

Comment:
Should these error messages specify "one or more"?

Reply:
Yeah, good point

Reply:
Updated in fa0079a